### DISCUSSION TITLE
Duda TPI3  - Coincidencias Parciales/Totales

### DISCUSSION THREAD

**Discussion by belencaponelli** [dds-jv/foro#284] [Consultas generales] [answered]
Buenas! Como va?

Estaba haciendo los Tests y me encontre con los siguientes dos:

- sePuedenListarMultiplesEventoEntreDosFechasParaUneUsuarieConCoincidenciaParcial()
- sePuedenListarMultiplesEventoEntreDosFechasParaUneUsuarieConCoincidenciaTotal()

No estoy segura si estoy entendiendo bien a lo que se refiere con coincidencia parcial y coincidencia total. Dejo en un ejemplo para  que me digan si esta bien el razonamiento:

- Evento 1 :  Comienza 10/05/2021 , Termina 15/05/2021
- Coincidencia Total: Seria que el evento se lista si las fechas indicadas son por ejemplo: inicio 08/05/2021, fin 20/05/2021
- Coincidencia Parcial: Seria que el evento se lista si las fechas indicadas son por ejemplo: inicio 08/05/2021, fin 10/05/2021

Espero que se haya entendido bien.

Gracias!

**Comment by flbulgarelli** [comment, marked as answer]
¡Hola!

Es correcto lo que decís. Aclaro también que la coincidencia parcial se daría aún si la fecha de fin del segundo evento fuera  el 11/05 por ejemplo. 

¡Saludos!
